java/ql/src/experimental/Security/CWE/CWE-073/FilePathInjection.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>External Control of File Name or Path, also called File Path Injection, is a vulnerability by which
+a file path is created using data from outside the application (such as the HTTP request, the database, or
+the filesystem). It allows an attacker to traverse through the filesystem and access arbitrary files.</p>
```

Comment:
```suggestion
a file path is created using data from outside the application (such as the HTTP request). It allows an attacker to traverse through the filesystem and access arbitrary files.</p>
```

Because your query uses `RemoteFlowSource`

Reply:
Changed